grader_service/grader_service/autograding/local_grader.py

```diff
@@ -293,32 +320,38 @@ async def _push_results(self):
         except CalledProcessError:
             pass
 
-        self.log.info(f"Creating new branch submission_{self.submission.commit_hash}")
+        self.log.info(f"Creating new branch "
+                      f"submission_{self.submission.commit_hash}")
         command = (
-            f"{self.git_executable} switch -c submission_{self.submission.commit_hash}"
+            f"{self.git_executable} switch -c "
+            f"submission_{self.submission.commit_hash}"
         )
         try:
             await self._run_subprocess(command, self.output_path)
         except CalledProcessError:
             pass
-        self.log.info(f"Now at branch submission_{self.submission.commit_hash}")
+        self.log.info(f"Now at branch "
+                      f"submission_{self.submission.commit_hash}")
 
         self.log.info(f"Commiting all files in {self.output_path}")
         try:
             await self._run_subprocess(
                 f"{self.git_executable} add -A", self.output_path
             )
             await self._run_subprocess(
-                f'{self.git_executable} commit -m "{self.submission.commit_hash}"',
+                f'{self.git_executable} commit -m '
+                f'"{self.submission.commit_hash}"',
                 self.output_path,
             )
         except CalledProcessError:
-            raise RuntimeError(f"Failed to commit changes")
+            raise RuntimeError("Failed to commit changes")
 
         self.log.info(
-            f"Pushing to {git_repo_path} at branch submission_{self.submission.commit_hash}"
+            f"Pushing to {git_repo_path} at branch "
```

Comment:
There's a small section in pep8 about whether binary operators should appear before or after a line break:
https://peps.python.org/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

It's not a hard rule, but suggests that it's easier to read if the operator appears as the first char on newline.